web_services/foxx/brightid/schemas.js

```diff
@@ -233,8 +233,8 @@ schemas = Object.assign({
 
   verificationGetResponse: joi.object({
     data: joi.object({
-      unique: joi.string().description("true if user is unique under given context"),
-      context: joi.string().description("the context name"),
+      unique: joi.string().description("true if user is unique under given application"),
+      application: joi.string().description("the application name"),
```

Comment:
`app: joi.string().description("the key of app"),`

Reply:
it's done